Examples/Python/tests/test_alignmentdecorator.py

```
@@ -0,0 +1,70 @@
+    alignmentStore = acts.examples.GeoIdAlignmentStore(geoIdMap)
+    alignDecoConfig.iovStores = [ ((10, 20), alignmentStore) ]
+    alignDecoConfig.garbageCollection = True
+    alignDecoConfig.gcInterval = 20
+
+    alignDeco = AlignmentDecorator(alignDecoConfig, acts.logging.DEBUG)
+
+    sequencer = Sequencer( events=100,
+        numThreads=4,
+        logLevel=acts.logging.INFO,
+    )
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)